domains/kelvin.json

```
@@ -0,0 +1,10 @@
+{
+    "description": "Dev Site",
+    "owner": {
+        "username": "kelvinelove",
+        "email": "kevypolu@yahoo.com"
+    },
+    "record": {
+        "CNAME": "https://kevhi.netlify.app"
```

CNAME can't contain protocol
```suggestion
        "CNAME": "kevhi.netlify.app"
```